data/PROJECTS.json

```diff
@@ -73,5 +73,13 @@
         "projectType": "Game",
         "award": "silver",
         "projectDescription": "Tether is an online game with support for all devices. It uses native Canvas graphics and smart animation techniques to build an all-out aesthetically pleasing and fun game!"
+    },
+    {
+        "repoName": "Brain-Controlled-Robotic-Joint",
+        "repoLogo": "",
```

Comment:
```suggestion

```
If you do not provide a repoLogo, leave the field out so that we can generate an image for you from Vercel.